src/components/MapEditor/__tests__/MapInteractive.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Map from '..';
+import MAP_OPTIONS from '../../../shared/services/configuration/map-options';
+
+describe('components/MapInteractive', () => {
+  const testLocation = {
+    geometrie: {
+      type: 'Point',
+      coordinates: [4, 52],
+    },
+  };
+
+  it('should render the map', () => {
+    const { container, getByText } = render(<Map options={MAP_OPTIONS} location={{}}/>);
+
+    // Map
+    expect(getByText('Leaflet')).toBeInTheDocument();
+
+    // Tile layer
+    expect(container.querySelector('.leaflet-tile-container')).toBeInTheDocument();
```

Comment:
There is no reference to this class name in the `MapEditor` component. If you need to verify element's existence, you can use the `data-testid` attribute on components and elements and assert `getByTestId` or `queryByTestId`